powerauth-restful-security-spring/src/main/java/io/getlime/security/powerauth/rest/api/spring/service/v2/ActivationService.java

```diff
@@ -90,7 +90,7 @@ public ActivationCreateResponse createActivation(ActivationCreateRequest request
 
             return response;
         } catch (Exception ex) {
-            logger.warn("Creating PowerAuth activation failed", ex);
+            logger.warn("Creating PowerAuth activation failed, error: {}", ex.getMessage());
             throw new PowerAuthActivationException();
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.